### PR TITLE
Add fox

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by jan-janssen** [comment]
The license is available at https://github.com/andreww/fox/blob/master/LICENSE but I am not sure about the SPDX identifier.

File: recipes/fox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - test -f ${PREFIX}/lib/libFoX_common.a
+    - test -f ${PREFIX}/lib/libFoX_wxml.a
+    - test -f ${PREFIX}/lib/libFoX_wkml.a
+    - test -f ${PREFIX}/lib/libFoX_sax.a
+    - test -f ${PREFIX}/lib/libFoX_dom.a
+
+about:
+  home: http://homepages.see.leeds.ac.uk/~earawa/FoX/
+  license: OTHER
+  license_family: OTHER
```
Comment (chrisburr) [inline recipes/fox/meta.yaml:34]: I think we've given up on this field now we're stricter about the content of `license:`
```suggestion
```

File: recipes/fox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - test -f ${PREFIX}/lib/libFoX_utils.a
+    - test -f ${PREFIX}/lib/libFoX_common.a
+    - test -f ${PREFIX}/lib/libFoX_wxml.a
+    - test -f ${PREFIX}/lib/libFoX_wkml.a
+    - test -f ${PREFIX}/lib/libFoX_sax.a
+    - test -f ${PREFIX}/lib/libFoX_dom.a
+
+about:
+  home: http://homepages.see.leeds.ac.uk/~earawa/FoX/
+  license: OTHER
```
Comment (chrisburr) [inline recipes/fox/meta.yaml:33]: It's a mix of Zlib  and BSD-3-Clause I think, in which case see: https://spdx.org/ids-how
```suggestion
  license: Zlib AND BSD-3-Clause
```

File: recipes/fox/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: dea0adc9cc035238fa9cdba42f2bf56481e3a64ac8aa0aece9119f127f71d4e7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - m4
```
Comment (chrisburr) [inline recipes/fox/meta.yaml:19]:
```suggestion
    - m4
    - make
```

**Comment by jan-janssen** [comment]
@chrisburr - Thanks a lot.

**Pull request merged**